### PR TITLE
Add linter

### PR DESCRIPTION
**Pull request by migibert**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -62,3 +63,4 @@ Vagrant.configure('2') do |config|
   config.vm.synced_folder '../ruggedpod-api', "/opt/ruggedpod-api"
   config.vm.provision 'shell', inline: $script, privileged: false
 end
+
```
Comment (ggiamarchi) [inline Vagrantfile]: Blank line to remove

File: package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "grunt-contrib-watch": "0.6.1"
+  },
+  "devDependencies": {},
+  "scripts": {},
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/RuggedPOD/ruggedpod-web.git"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (ggiamarchi) [inline package.json]: That's wrong

File: package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "grunt-contrib-jshint": "0.11.0",
+    "grunt-contrib-watch": "0.6.1"
+  },
+  "devDependencies": {},
+  "scripts": {},
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/RuggedPOD/ruggedpod-web.git"
+  },
+  "author": "",
```
Comment (ggiamarchi) [inline package.json]: We should put a value or remove the key

File: package.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  "version": "1.0.0",
+  "description": "This is a management console built on top of the [RuggedPOD API](https://github.com/RuggedPOD/ruggedpod-api).",
+  "main": "Gruntfile.js",
+  "directories": {
+    "doc": "doc"
+  },
+  "dependencies": {
+    "grunt": "0.4.5",
+    "grunt-contrib-jshint": "0.11.0",
+    "grunt-contrib-watch": "0.6.1"
```
Comment (ggiamarchi) [inline package.json]: Those 3 dependencies are more likely to be in the devDependencies section

File: package.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "ruggedpod-web",
+  "version": "1.0.0",
```
Comment (ggiamarchi) [inline package.json]: It should match the `bower.json` version. Currently `0.0.1`.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+  "directories": {
+    "doc": "doc"
+  },
+  "dependencies": {
+    "grunt": "0.4.5",
+    "grunt-contrib-jshint": "0.11.0",
+    "grunt-contrib-watch": "0.6.1"
+  },
+  "devDependencies": {},
+  "scripts": {},
```
Comment (ggiamarchi) [inline package.json:15]: empty = useless :crying_cat_face:

File: package.json (hunk 4 above)
Comment (ggiamarchi) [inline package.json]: Actually `grunt-contrib-watch` is useless here.

File: Gruntfile.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+      }
+    },
+    watch: {
+      files: ['<%= jshint.files %>'],
+      tasks: ['jshint']
+    }
+  });
+
+  grunt.loadNpmTasks('grunt-contrib-jshint');
+  grunt.loadNpmTasks('grunt-contrib-watch');
```
Comment (ggiamarchi) [inline Gruntfile.js]: `grunt-contrib-watch` is useless here.

**Comment by ggiamarchi** [comment]
@migibert 
- The branch `ci` have an unclean history (it looks like there were multiple rebases done on this branch and it introduced useless change sets). So, i have cherry-pick the relevants commits (everything about `.travis.yml`) here.
- I have pushed a commit that fix all the lint errors. Build is now green :smile_cat:

**Comment by maxiterr** [comment]
LGTM

**Pull request merged**